src/views/createCommunity/index.tsx

```diff
@@ -518,11 +526,19 @@ function CreateCommunityScreen() {
                         setSending(false);
                         setSendingSuccess(true);
                     } else {
+                        Clipboard.setString(
```

Comment:
Why are we copying the error to the clipboard?

Reply:
It's useful to create tickets and include the error. We need to add some message saying that it was copied to the clipboard. Something like a https://callstack.github.io/react-native-paper/snackbar.html
Also https://impactmarket.atlassian.net/browse/IPCT1-245